app/newsfeed/server/initialize.js

```diff
@@ -0,0 +1,19 @@
+// import { Meteor } from 'meteor/meteor';
+// import { settings } from '../../settings';
+// import { Mongo } from 'meteor/mongo';
+
+import { createRoom } from '../../lib/server/functions';
+import { Users, Rooms } from '../../models';
+
+function initializeNewsfeed() {
+	const usersArray = Users.findByUsername().fetch();
+	usersArray.forEach((user) => {
+		if (user.type === 'user') {
+			if (!Rooms.findOneByName(`news_${ user._id }`)) {
```

Comment:
To optimise this and similar checks.. check the following links,

https://stackoverflow.com/questions/31663037/given-a-list-of-ids-whats-the-best-way-to-query-which-ids-do-not-exist-in-the
https://blog.serverdensity.com/checking-if-a-document-exists-mongodb-slow-findone-vs-find/
https://stackoverflow.com/questions/8389811/how-to-query-mongodb-to-test-if-an-item-exists

Reply:
I'll implement it and take care of it next time. Thank you bhaiya